README.md

```diff
@@ -18,7 +18,7 @@ $ composer require tanigami/specification
 
 ```php
 <?php
-
+require 'vendor/autoload.php';
```

Comment:
If I get your point correctly, actually this composer autoload file manually required it is expected to be in other place, or in any place you deside into your project.
I think there is no need to clarify this basic point, well explained into the [composer website](https://getcomposer.org/doc/01-basic-usage.md#autoloading).